webapp/index.html

```diff
@@ -8,7 +8,7 @@
 
 	<script id="sap-ui-bootstrap" type="text/javascript"
 		src="resources/sap-ui-core.js"
-		data-sap-ui-theme="sap_bluecrystal"
+		data-sap-ui-theme="sap_belize"
```

Comment:
As this project consumes local dependencies the [`bower.json`](https://github.com/SAP/openui5-sample-app/blob/master/bower.json) and [`Gruntfile.js`](https://github.com/SAP/openui5-sample-app/blob/master/Gruntfile.js) files need to be adopted accordingly.